includes/configure/replace.yaml

```diff
@@ -33,64 +32,17 @@
       dest: "/tmp/{{ vyos_config_temp_config_file }}"
 
   - block:
-    - name: enter configuration mode
-      cli:
-        command: configure
-
     - name: replace currect running configuration
-      cli:
-        command: "load /tmp/{{ vyos_config_temp_config_file }}"
+      cli_config:
+        config: "load /tmp/{{ vyos_config_temp_config_file }}"
```

Comment:
This is not a config command on vyos. Please keep `cli` isntead.

Reply:
@trishnaguha I am not sure if I fully understand this. I have pushed some new commits to further support this. I understand that cli_config is intended for pushing text based configuration into network devices, however, having the load command executed using cli_config simplifies the replace task, because (a) we don't need to execute configure and exit separately using loop (b) cli_config supports commit_comment option and hence (c) we don't need separate tasks/checks for commit operations.

Reply:
@NilashishC Sorry about that. I missed the `configure` part in the task.